### PR TITLE
Add indicators section for Servo Board

### PR DESCRIPTION
**Pull request by trickeydan**
The indicators section is currently not there in the servo board docs. I have created a table for `j5` (developer docs) purposes and thought it'd be useful for students also.

Fixes https://github.com/srobo/docs/issues/90

### CODE REVIEW

File: kit/servo_board.md [hunk 1]
```diff
@@ -16,14 +16,17 @@ Board Diagram
...
-| Power           | The board is powered   | On
--->
+| LED           | Meaning                                                                                          | Initial power-up state |
+| ------------- | ------------------------------------------------------------------------------------------------ | ---------------------- |
+| Power         | The board is powered over USB.                                                                   | On                     |
+| 5.5V On       | There is 5.5V power on the board. This usually indicates that the 12V connector rail is powered. | Off                    |
+| Aux On        | There is auxiliary power on the board.                                                           | Off                    |
+| Status        | The board has successfully booted.                                                               | On                     |
+| Error         | An error has occurred. This often indicates a disconnection of the 12V connector.                | Off                    |
+| Outputs 00-11 | Currently unimplemented in firmware.                                                             | Off                    |
```
Comment (RealOrangeOne) [inline kit/servo_board.md]: Not sure how useful it is to document LEDs which are never used. 
```suggestion
```

Comment (trickeydan) [reply kit/servo_board.md]: Ah I meant to remove that. I think it's useful to have that in dev docs though, as hopefully they'll be implemented one day.

File: kit/servo_board.md [hunk 2]
```diff
@@ -16,14 +16,16 @@ Board Diagram
...
 
-|   LED           | Meaning                | Initial power-up state
-|-----------------|------------------------|----------------------
-| Power           | The board is powered   | On
--->
+| LED           | Meaning                                                                                          | Initial power-up state |
+| ------------- | ------------------------------------------------------------------------------------------------ | ---------------------- |
+| Power         | The board is powered over USB.                                                                   | On                     |
+| 5.5V On       | There is 5.5V power on the board. This usually indicates that the 12V connector rail is powered. | Off                    |
+| Aux On        | There is auxiliary power on the board.                                                           | Off                    |
```
Comment (PeterJCLaw) [inline kit/servo_board.md]: What is "Aux On"? This doesn't seem to be included on the diagram. Should we be updating the diagram?

File: kit/servo_board.md [hunk 3]
```diff
@@ -16,14 +16,16 @@ Board Diagram
...
 ----------
 
-|   LED           | Meaning                | Initial power-up state
-|-----------------|------------------------|----------------------
-| Power           | The board is powered   | On
--->
+| LED           | Meaning                                                                                          | Initial power-up state |
+| ------------- | ------------------------------------------------------------------------------------------------ | ---------------------- |
+| Power         | The board is powered over USB.                                                                   | On                     |
+| 5.5V On       | There is 5.5V power on the board. This usually indicates that the 12V connector rail is powered. | Off                    |
```
Comment (PeterJCLaw) [inline kit/servo_board.md]: What is "5.5V On"? This doesn't seem to be included on the diagram. Should we be updating the diagram?

File: kit/servo_board.md [hunk 4]
```diff
@@ -16,14 +16,16 @@ Board Diagram
...
+| Aux On        | There is auxiliary power on the board.                                                           | Off                    |
+| Status        | The board has successfully booted.                                                               | On                     |
+| Error         | An error has occurred. This often indicates a disconnection of the 12V connector.                | Off                    |
```
Comment (PeterJCLaw) [inline kit/servo_board.md]: What is "Error"? This doesn't seem to be included on the diagram. Should we be updating the diagram?

File: kit/servo_board.md (hunk 2 above)
Comment (trickeydan) [reply kit/servo_board.md]: Yes the diagram needs updating by the look of things. The LEDs are definitely on the board if you look at one. They are marked on the sticker on the casing as "5.5V On", "Aux On", "Error".

File: kit/servo_board.md (hunk 3 above)
Comment (trickeydan) [reply kit/servo_board.md]: Yes the diagram needs updating by the look of things. The LEDs are definitely on the board if you look at one. They are marked on the sticker on the casing as "5.5V On", "Aux On", "Error".

File: kit/servo_board.md (hunk 4 above)
Comment (trickeydan) [reply kit/servo_board.md]: Yes the diagram needs updating by the look of things. The LEDs are definitely on the board if you look at one. They are marked on the sticker on the casing as "5.5V On", "Aux On", "Error".

**Comment by kierdavis** [comment]
This table looks correct to me.

I agree that the diagram should be updated so that it shows the LEDs that are documented in the table, and no others. I think the labels in the diagram should also be updated to better correspond to those used in the table and those used on the physical board because there is currently potential for confusion: the LED labelled "power supply status" on the diagram is the 5.5V supply indicator, while the LED to the one labelled "power" in the table is the 3.3V supply indicator.

I have no preference whether this is done as an extension to this PR or if we create a separate issue or PR to track it.

**Comment by PeterJCLaw** [comment]
I've updated the SVG board diagram and annotated the docs table with the hardware names of the LEDs for easier cross referencing with our [internal docs](https://studentrobotics.org/runbook/kit/hardware/kit-debugging/#servo-board).

It would be great if we could work out where the "Error" LED is and add that to the diagram. Once that's done we can regenerate the PNG diagram too.

File: kit/servo_board.md (hunk 4 above)
Comment (PeterJCLaw) [reply kit/servo_board.md]: https://github.com/srobo/docs/pull/103/commits/de55d345964825f9469f5280f746243c64e3af0b

**Comment by PeterJCLaw** [comment]
> It would be great if we could work out where the "Error" LED is and add that to the diagram. Once that's done we can regenerate the PNG diagram too.

Done, via this reference photo: https://studentrobotics.slack.com/archives/CC8EB6YMQ/p1664723008488329?thread_ts=1664719657.248689&cid=CC8EB6YMQ

I've also added the Aux On LED to the diagram which I missed earlier.

File: kit/servo_board.md [hunk 5]
```diff
@@ -16,14 +16,15 @@ Board Diagram
...
-|   LED           | Meaning                | Initial power-up state
-|-----------------|------------------------|----------------------
-| Power           | The board is powered   | On
--->
+| LED                   | Meaning                                                                                          | Initial power-up state |
+| --------------------- | ------------------------------------------------------------------------------------------------ | ---------------------- |
+| Power <!-- DS1 -->    | The board is powered over USB.                                                                   | On                     |
+| 5.5V On <!-- DS15 --> | There is 5.5V power on the board. This usually indicates that the 12V connector rail is powered. | Off                    |
+| Aux On <!-- DS16 -->  | There is auxiliary power on the board.                                                           | Off                    |
+| STATUS\|ERROR <!-- DS10 --> | Blue when the board has successfully booted<br />Flickering red when changing servo positions<br />Solid pink if an error has occurred (often 12V power being lost) | Blue |
```
Comment (WillB97) [inline kit/servo_board.md:27]: What is this flickering refering to? Looking at the frmware this is only during the interrupt routine which under normal operation should not be on long enough to be visible.

Comment (PeterJCLaw) [reply kit/servo_board.md:27]: Ah, ok. I'd assumed it was going to be visible.

Comment (PeterJCLaw) [reply kit/servo_board.md]: 76ba883c590a3d0779899124d57c81efbefde2d5

**Review by WillB97 — APPROVED** [review]

**Pull request merged**